### ISSUE TITLE
Change of big seek behaviour for skip ahead / back.

### ISSUE THREAD

**Issue by danjames92** [plexinc/plex-for-kodi#99]
The standard behaviour for Plex Media Player and some other clients such as Android TV is 30 seconds forward, 10 or 15 seconds back. 

With the current iteration, you move forward or backwards 10 seconds at a time and you have to confirm the action before it takes place. 

This means if I want to skip forward one minute I have to press a button 6 times to skip forward 60 seconds then another to confirm. With Plex Media Player, I can quickly skip forward by 1 minute by pressing 2 buttons.

I would love to see Big Seek follow Plex Media Player's standard step moving forward. Thanks for consideration.

**Comment by joggs** [comment]
I agree, alternatively implementing the current kodi behavior with is increasing skip steps;10,30,60sec and if if idle for 500-750ms the skip size is submitted, instead of click submit which is an unnecessary extra click.

**Comment by keithah** [comment]
As stated, seeking is a limitation imposed by how Kodi handles streaming mkv, and because we have a desire to have the same user interface for direct play and transcoding (to not confuse less educated users) this is the current compromise.

We can revisit this when we have time to see if there has been improvements that we can utilize (possibly MPEG-DASH support, or something of that nature)

**Comment by joggs** [comment]
Great. Looking forward to it.
Out of curiosity, how is not the 10 second limit and submit as a click() instead of a 500ms timeout just a UI adjustment? Don't you just post the new selected playback position to the server in all different cases?
